.github/workflows/docs.yaml

```
@@ -24,6 +23,9 @@ jobs:
             python-sphinx \
             texinfo \
             # EOF
-      - uses: actions/checkout@v2
-      - name: Build
+      - uses: actions/checkout@v4
+        with:
+          persist-credentials: false
+          submodules: recursive
```

```suggestion
      - uses: actions/checkout
```

The action settings don't matter for us, no real point in specifying them.

---

I am not comfortable removing `persist-credentials`. This is a recipe for being bitten in the future with how a modern `actions/checkout` behaves, for example if we want to make this workflow one day publish the documentation somewhere.

https://github.com/actions/checkout/issues/485